pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/StudySql.java

```diff
@@ -0,0 +1,21 @@
+package org.broadinstitute.ddp.db.dao;
+
+import org.jdbi.v3.sqlobject.SqlObject;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+
+public interface StudySql extends SqlObject {
```

Comment:
Do we want this to be StudyInviteSettingSql rather than StudySql !! 
OR Planning to eventually move such study related DB stuff into StudySql.java ??

Reply:
I think we have way too many of these jdbi/dao files so I wanted to consolidate them more. Besides, it's just a few queries and it's study-related, so I think it's okay to put into one file/interface.